components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/DataFlowProcessingEngine.cs

```diff
@@ -159,6 +161,39 @@ public class DataFlowProcessingEngine : IProcessingEngine, IDisposable {
             return Task.CompletedTask;
         }
 
+        /// <inheritdoc/>
+        public Task<JobDiagnosticInfoModel> GetDiagnosticInfo() {
+            var totalSeconds = (DateTime.UtcNow - _diagnosticStart).TotalSeconds;
```

Comment:
Looks like this also needs a condition on `_diagnosticStart != DateTime.MinValue` similar to `totalDuration`, no ?

Reply:
`totalSeconds ` is used only if `_diagnosticStart != DateTime.MinValue` then I don't think we need that condition.. It's redundant